schema.sql

```diff
@@ -6,6 +6,7 @@ CREATE TABLE public.vet_clinic
     escape_attempts integer NOT NULL,
     neutered boolean NOT NULL,
     weight_kg numeric NOT NULL,
+    species text,
```

Comment:
- [ ] Editing already ran migration is not a recommended way to add new fields actually, you should use this statement `ALTER TABLE animals ADD species varchar(255);` instead!